src/components/privatePlaylist.vue

```diff
@@ -0,0 +1,67 @@
+<script setup lang="ts">
+import { ref, Ref } from "vue";
+import {
+  TrackCollectionDetails,
+  TrackCollectionApi,
+} from "@bcc-code/bmm-sdk-fetch";
+
+const collections: Ref<TrackCollectionDetails[]> = ref([]);
+
+new TrackCollectionApi()
+  .trackCollectionGet()
+  .then((r) => {
+    collections.value = r;
+  })
+  .catch(() => {});
+</script>
+
+<template>
+  <div>
+    <h3>Playlists</h3>
+    <ul>
+      <li>
+        <RouterLink
+          v-for="collection in collections"
```

Comment:
With these changes, we have one unordered list (which is good) but this one list has also just only one list-item, which contains the many playlists the user might have. Each playlist should (as a list has multiple list-items) be on a separate list-item.